app/Http/Controllers/V2/User/IndexMyActionsController.php

```diff
@@ -22,15 +22,15 @@ public function __invoke(Request $request): AnonymousResourceCollection
             ->with('targetable')
             ->pending();
 
+        $qry->where('project_id', $projectIds);
```

Comment:
I'm curious about this change; wasn't the previous approach more correct? If there are multiple project ids, this call would need to be `whereIn` as on line 29.